<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEquity">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRefresh">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1S</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5S</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10S</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30S</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1H</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelRefresh">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>111</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpacity">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Opacity</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
